packages/liveblocks-node/src/client.ts

```diff
@@ -266,22 +849,18 @@ export class Liveblocks {
   }): Promise<ThreadParticipants> {
     const { roomId, threadId } = params;
 
-    const resp = await this.get(
+    const res = await this.get(
       `/v2/rooms/${encodeURIComponent(roomId)}/threads/${encodeURIComponent(
         threadId
       )}/participants`
     );
 
-    const body = await (resp.json() as Promise<ThreadParticipants>);
-
-    if (resp.status !== 200) {
-      throw {
-        status: resp.status,
-        ...body,
-      };
+    if (!res.ok) {
+      const text = await res.text();
+      throw new ApiError(res.status, text);
```

Comment:
Could this generic error handling not be part of the `.get()`, `.post()`, etc methods? It looks like this needs to be handled the same everywhere?